lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'menu'
+require_relative 'sendsms'
+
+class TakeAway
+
+  attr_reader :menu, :order
+
+  def initialize (menu = Menu.new)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)